lib/money/money.rb

```diff
@@ -266,8 +270,10 @@ def all_rational?(splits)
   def value_to_decimal(num)
     if num.respond_to?(:to_d)
       num.is_a?(Rational) ? num.to_d(16) : num.to_d
-    else
+    elsif num.is_a?(Money) || num.is_a?(Numeric) || num.is_a?(String)
```

Comment:
When would we have `num.is_a?(Numeric) || num.is_a?(String)`? Both respond to the `.to_d` in theory